examples/advanced_operations/add_sitelink.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Demonstrates how to create a sitelink extension."""
+
+from __future__ import absolute_import
+
+import argparse
+import six
+import sys
+import uuid
```

Comment:
This appears to be unused, can you remove it?

Reply:
Hi Ben, thx for the feedback. I am not a 100% sure how you typically want to run those examples. Judging from the other examples though, it seems like they are supposed to be run from the command line.
This means we need this part to get the customer_id.
Which brings me to another question, I just realized is somewhat strange. Actually the client gets instanciated with a ```client_customer_id```. However, it seems like its not being used or being exposed for that matter.

Reply:
Hi @FBosler thanks for making the updates! I'm definitely planning to make the style changes to the other examples at some point. If you have time and are interested please feel free to submit a PR but don't feel obligated.

The examples are indeed designed to be run from the command line. Since there can be variance in the CID and also inputs such as keywords, etc. it makes them much easier to work with. 

You're correct about the `client_customer_id` in that the client doesn't import it. In the client library for the AdWords API there was an option to include the CID in yaml, but that option doesn't exist in this library so you need to supply it explicitly. Is having the option to include it in yaml something you'd be interested in having? If so I can add a feature request and see if it's worth adding across the libraries.

Reply:
Actually @FBosler I created this issue to continue the discussion about CID so that I can merge this PR. https://github.com/googleads/google-ads-python/issues/new